src/App.vue

```diff
@@ -94,69 +39,18 @@
-                  </button>
-                  <button type="button" class="main-balls__content main-balls__content--blue">
-                    <span class="main-balls__gift"></span>
-                    <span class="main-balls__text">Подарить балы</span>
-                  </button>
-                </div>
+                <Balls></Balls>
                 <div class="main-content-choise">
-                  <button :id="button.id" v-for="button in buttons" :key="button.id"
-                    type="button"
-                    class="main-content-choise__all
-                    js__choise-btn" @click="choise = button.id; categoryValue(choise)"
-                    :class="{choise_active:choise === button.id}"
-                  >
-                    {{button.name}}
-                  </button>
+                <Buttons :valueId="category" @categoryValue="changeCategory"></Buttons>
                 </div>
               </div>
             </div>
             <!-----------------  SECTION MAIN CARD --------------- -->
             <main class="main-content-items main-indent">
               <div class="main-content__items js__catalog">
-                <div class="main-block" v-for="item in products"
-                  :key="item.id"
-                >
-                  <section class="main-block__item">
-                    <div class="main-card">
-                      <img :src="item.img"
-                      alt="Картинка футболки" class="main-card__img"
-                        width="330" height="330">
-                        <span class="main-card__badge" v-if="item.isNew">new</span>
-                        <span class="main-card__badge" v-else></span>
-                    </div>
-                    <div class="main-card__description">
-                      <span class="main-card__description-price">
-                      {{item.price}} баллов
-                      </span>
-                      <h3 class="main-card__description-title">
-                      {{item.title}}
-                      </h3>
-                      <span class="main-card__description-hint" v-if="item.isClothes">
-                      {{item.label}} {{item.volume}}
-                      </span>
-                      <span class="main-card__description-hint" v-else>
-                      {{item.label}} {{item.volume}}
-                      </span>
-                    </div>
-                    <div class="main-card__description-hidden"
-                      >
-                      <button class="main-card__description-btn" type="button"
-                      @click="showModal()">
-                      Заказать
-                      </button>
-                    </div>
-                  </section>
-                  </div>
+                <MainCards :data="item" :id="item.id"
+                v-for="item in products" :key="item.id"
+                @open="openCard(item)"></MainCards>
```

Comment:
порядок атрибутов поправь 

и пусти их построчно - так будет проще читать

```suggestion
           <MainCards 
                v-for="item in products" 
                :key="item.id"
                :data="item" 
                :id="item.id"
                @open="openCard(item)"
            ></MainCards>
```

Reply:
Согласен